Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -21,16 +21,23 @@ function SetTaskContextVariable
     param([string][ValidateNotNullOrEmpty()]$varName, 
           [string]$varValue)
         
-    #[Environment]::SetEnvironmentVariable($varName, $varValue)
+    [Environment]::SetEnvironmentVariable($varName, $varValue)
     Write-Host "##vso[task.setvariable variable=$varName;]$varValue"
 }
 
 function GetTaskContextVariable()
 {
 	param([string][ValidateNotNullOrEmpty()]$varName)
         
-    #return ([Environment]::GetEnvironmentVariable($varName))
-	return Get-TaskVariable -Context $distributedTaskContext -Name $varName    
+    $value = [Environment]::GetEnvironmentVariable($varName);
```

Comment:
The library function (Get-TaskVariable) does not read the variable that was set in the current build task. It reads only vars set in the previous build tasks. So I created a wrapper that adds this functionality.